### ISSUE TITLE
How to read Temperature and Humidity from myCO2?

### ISSUE THREAD

**Issue by shtrom** [Sensirion/myAmbienceManual#1]
tl;dr: How can I access temperature and humidity information from a myCO2 module in myAmbience Android app?

Hi,

For lack of a better repository, I'm asking the question here. Please feel free to redirect me to a better location for the question.

I have a myCO2 gadget, which is working well. I know from hooking it up to Home Assistant that it can report temperature and humidity (albeit sometimes with the wrong sigen) in addition to CO2 level.

I was hoping to see this in the myAmbience app too, but as I tested this out today, I found no way to access this information.

How can I access this information?

**Comment by psachs** [comment]
The humidity and temperature values of the SCD4x sensor in the myCO2 are not exposed in MyAmbience, since they do not represent the ambient temperature and humidity values, but the actual measured values which are potentially heavily influenced by internal and external temperature sources (since the myCO2 is directly connected to a USB port).

Therefore we decided not to show those values in MyAmbience. We believe those values are not valuable to the user and will potentially confuse the users, since they are wrong if interpreted as ambient temperature and humidity.

I hope this clarifies things.

**Comment by shtrom** [comment]
Hi Pascal,

Yes, this makes sense.

In my case, I did not notice deviations between the MyCO2 and other climate sensors closeby, but fair enough.

In any case, if there's a chance for this to be added as some sort of advanced mode, pleas put my name down on the waitlist 😁

Thanks for the quick reply!
-- 
Olivier Mehani ***@***.***>
Sent from my mobile, please excuse my brevity.
